AntennaTracker/Parameters.h

```diff
@@ -92,6 +92,9 @@ class Parameters {
         k_param_gcs3,               // stream rates for fourth MAVLink port
         k_param_log_bitmask,        // 140
         k_param_notify,
+#if HAL_WITH_UAVCAN
+        k_param_BoardConfig_CAN,
```

Comment:
I just noticed this one can't be here, it will move the parameter indexes below - which can never happen.